tools/ping/src/lib.rs

```diff
@@ -392,15 +412,11 @@ async fn ping_via_node(
         chain_id,
         genesis_hash,
         head_height,
-        time::Duration::seconds(recv_timeout_seconds.into()),
-    )
-    .await
-    {
+        time::Duration::seconds(recv_timeout_seconds.into())).await {
         Ok(p) => p,
         Err(ConnectError::HandshakeFailure(reason)) => {
             match reason {
-                HandshakeFailureReason::ProtocolVersionMismatch { version, oldest_supported_version } => anyhow::bail!(
-                    "Received Handshake Failure: {:?}. Try running again with --protocol-version between {} and {}",
+                HandshakeFailureReason::ProtocolVersionMismatch { version, oldest_supported_version } => anyhow::bail!( "Received Handshake Failure: {:?}. Try running again with --protocol-version between {} and {}",
```

Comment:
intentional?

Reply:
Nope, thanks for spotting.
I trust `cargo fmt` too much.